src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php

```diff
@@ -73,10 +83,18 @@ public function renderSeoTags(array $seoExtension, array $content, array $urls,
             $webspaceKey = $webspace->getKey();
         }
 
+        $request = $this->requestStack->getCurrentRequest();
```

Comment:
Shouldn't be a problem as far as I can tell. But what about generating protocol relative URLs? (`//...`)